tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1409,9 +1409,8 @@ function memberOutside() {
 
 function memberInAndOutWithCalls() {
   return (
-    a
```

Comment:
This changes the meaning of the text, can you add a few more characters here?

Reply:
Added.